myhpom/templates/myhpom/content/why-plan.html

```diff
@@ -0,0 +1,237 @@
+{% extends "myhpom/base.html" %}
+{% load scribbler_tags %}
+{% load staticfiles %}
+
+{% block main_container_class %}myhpom-page__container{% endblock main_container_class %}
+
+{% block main_content %}
+    <section class="content-page__hero-banner" style="background-image: url({% static 'myhpom/img/why-plan/father-at-home-with-tablet-video-chatting_web.jpg' %})"></section>
+    <section class="why-plan-overview">
+        <div class="container">
+            <div class="row">
+                <div class="col-12">
+                    <h1 class="why-plan-overview__title">
```

Comment:
Should the title and subtitle be scribbleable along with the body? So throughout.